lib/OpenQA/Schema/ResultSet/Jobs.pm

```diff
@@ -225,10 +227,34 @@ sub create_from_settings {
     return $job;
 }
 
+sub rg_files {
+    my ($self, $dir, $regexp) = @_;
+    my @ret;
+    my $res = run_cmd_with_log_return_error(
+       ['rg', '--type','perl', '--type', 'py', '-l', qq('$regexp'), $dir]);
```

Comment:
In https://progress.opensuse.org/issues/96058 we have considered `git grep` which would work without additional dependencies. I am ok with trying rg as an option but so far I still consider git a better alternative